merge-dev-to-master-v102.0

hitz opened the pull request.

forresttanaka reviewed (approved):
Looks great!

forresttanaka reviewed (approved):
Left a couple of comments mostly for future ticket code changes. Also, we really need to clean up DatasetComponent, but that’s not for this ticket.

forresttanaka commented:
Just noticed it went back to In Progress, but I’ll submit what I have so far.

forresttanaka committed:
ENCD-6157-allow-multiple-types-for-report-view-button (#4122)

forresttanaka committed:
ENCD-5893-cart-reclassify-non-raw-files-without-assembly-as-processed (#4074)

khzlin committed:
ENCD-6189-add-experimental-input-UI (#4140)

khzlin committed:
ENCD-6168-update-fc-series-search-title (#4142)

jenjou committed:
ENCD-6195 Add datapoint to Transgenic Enhancer Experiment (#4139)

keenangraham committed:
SNO-212-support-gzip-attachment (#4126)

bonitalam committed:
ENCD-6131-update-donor-ethnicity-enum (#4143)

forresttanaka committed:
ENCD-6199-fixed-deep-biosample-crash (#4141)

keenangraham committed:
SNO-210-read-columns-from-configs (#4114)

forresttanaka committed:
ENCD-6175-support-collection-series-and-transgenic-enhancer-experiments-in-cart (#4138)

forresttanaka committed:
SNO-214-upgrade-node-to-version-12-for-circleci-tests (#4136)

iyoungworth committed:
ENCD-6194-relax-unreplicated-audit-on-long-read-scRNA-seq (#4150)

jenjou committed:
ENCD-6145-more-facet-headers (#4129)

bonitalam committed:
ENCD-6201-add-purpose-treatment-UI (#4147)

emro committed:
ENCD-4287 update d3 (#4144)

emro committed:
ENCD-6187 fix legend for ccres (#4151)

jenjou committed:
ENCD-6182-scrna-output-types (#4145)

zoldello committed:
ENCD-6117-cell-line-matrix (#4133)

iyoungworth committed:
ENCD-6178-library-property-updates-Yu-lab (#4146)

forresttanaka committed:
ENCD-6206-appended-exists-facet (#4155)

iyoungworth committed:
ENCD-6196-examined-loci-dependency-include-readout (#4154)

jenjou committed:
ENCD-6192-cart-schema-updates (#4135)

zoldello committed:
ENCD-6205-fix-browser-track-in-region-search (#4152)

jenjou committed:
ENCD-6202-add-enums-for-restriction-enzymes (#4153)

keenangraham committed:
ENCD-6207-remove-columns-from-dataset-schema (#4156)

lrowe committed:
ENCD-6197-devcontainer (#4137)

khzlin committed:
ENCD-6183-hic-scatac-qc (#4149)

forresttanaka committed:
ENCD-6179-fcc-menu (#4157)

gabdank committed:
ENCD-6208 remove control (#4161)

keenangraham committed:
SNO-217-register-search-config-pieces-from-item (#4160)

gabdank committed:
ENCD-6209-updating-FCS-assay-title (#4164)

gabdank committed:
update-snovault-1.0.62 (#4165)

zoldello committed:
ENCD-6220-deeply-profile-missing-facets (#4171)

forresttanaka committed:
ENCD-6217-report-list-crash (#4169)

ottojolanki committed:
ENCD-6223 pin responses to 0.14.0 (#4173)

gabdank committed:
REVERT ENCD-6179 (#4172)

jenjou committed:
ENCD-6218 Add facets and facet groups to donor  (#4170)

paul-sud committed:
ENCD-6224-fix-hicqualitymetric-dependencies (#4174)

gabdank committed:
update-version-v124.0 (#4177)

paul-sud committed:
ENCD-6226-remove-git-protocol (#4176)

jenjou committed:
ENCD-6222-gkm-outputs (#4182)

zoldello committed:
ENCD-5821-genome-browser-mislabels-hg19 (#4166)

keenangraham committed:
ENCD-6210-point-to-new-rnaget-endpoints (#4183)

forresttanaka committed:
ENCD-6191-listed-carts (#4184)

cricketsloan committed:
ENCD-6114 new assay slims (#4179)

gabdank committed:
ENCD-6232 assay title (#4186)

zoldello committed:
ENCD-6225 valis track not refreshed (#4181)

iyoungworth committed:
ENCD-6088-Series-type-and-expression-property-for-sorted-ATAC-datasets (#4185)

jenjou committed:
ENCD-6216-gm-updates (#4180)

iyoungworth committed:
ENCD-5890-adjust-pulse-chase-time-series-page (#4189)

* ENCD-5890-adjust-PulseChaseTimeSeries-page

continuing-work

minor-change

more-inserts-for-UI-testing

adjustment-to-bdd-test

addressing-code-review

add-comma-sep-within-dataset

checking-redundant-complete-entries-in-search-UI

* fixed-breadcrumbs

gabdank committed:
ENDC-6230 FCC calculated properties (#4178)

jenjou committed:
ENCD-6227-immune-slims (#4188)

jenjou committed:
ENCD-6244 CRISPR Method Dependency (#4194)

jenjou committed:
ENCD-6246-add-missing-slims (#4195)

gabdank committed:
ENCD-6247 adding control back (#4196)

gabdank committed:
ENCD-6250 audit readout (#4201)

forresttanaka committed:
ENCD-6252-circleci-js-memory (#4205)

hitz committed:
Update to v125.0 (#4206)

jenjou committed:
ENCD-6237-fix-search-title-biosample-summary (#4200)

mingjiecn committed:
ENCD-6240 antibody page not rendering (#4197)

forresttanaka committed:
ENCD-6152-cart-current (#4191)

khzlin committed:
ENCD-6233-experiment-sex-facet (#4198)

mingjiecn committed:
ENCD-6245 add user (#4193)

lrowe committed:
ENCD-6242 For AWS_* env vars to work use None instead of 'default' (#4192)

mingjiecn committed:
ENCD-6251 add aliases for charles-lee namespace (#4208)

emro committed:
ENCD-6229 fix unnamed Valis tracks (#4217)

iyoungworth committed:
ENCD-6211 add-inclusion-list-reference-property-to-library (#4210)

ottojolanki committed:
ENCD-6258 add -Dlog4j2.formatMsgNoLookups=true to jvm.options (#4219)

gabdank committed:
ENCD-6256-scatac (#4213)

gabdank committed:
ENCD-6228 gene dependency update (#4209)

emro committed:
ENCD-5946 immune cells rebased (#4211)

jenjou committed:
ENCD-6253-more-immune-slims (#4215)

mingjiecn committed:
ENCD-6239 segway QC (#4222)

jenjou committed:
ENCD-6255-culture-date-audit (#4216)

khzlin committed:
ENCD-6234-add-description-fcs-search (#4199)

gabdank committed:
ENCD-6249 ukbb (#4214)

khzlin committed:
ENCD-6235-adjust-fcs-table (#4202)

mingjiecn committed:
ENCD-6248 qc for sc rna processing (#4207)

forresttanaka committed:
ENCD-6213-redesigned-home-page (#4220)

zoldello committed:
ENCD-6070-human-donor-matrix (#4167)

forresttanaka committed:
ENCD-6104-series-table (#4226)

iyoungworth committed:
ENCD-6265-add-BruLibraryQualityMetric-object (#4229)

jenjou committed:
ENCD-6263 Add slim for Ribo-seq (#4227)

iyoungworth committed:
ENCD-6260-output-type-functional-conservation-mapping (#4225)

forresttanaka committed:
ENCD-6266-add-immune-cells-and-human-donors-to-home-page (#4231)

ottojolanki committed:
ENCD-6261 nolookups to jvm options (#4223)

gabdank committed:
update-version-v126.0 (#4236)

mingjiecn committed:
ENCD-6016 fix bad filters (#4224)

mingjiecn committed:
ENCD-6257 fastq dependency fix (#4230)

forresttanaka committed:
ENCD-6271-award-labels (#4238)

gabdank committed:
ENCD-6270-indexed-reads (#4233)

khzlin committed:
ENCD-6264-display-annotation-trait (#4242)

emro committed:
ENCD-6269 pin Valis summary page location (#4241)

mingjiecn committed:
ENCD-6098 update visualize limit for matrix and search (#4234)

iyoungworth committed:
ENCD-6241-add-3D-structure-output-type (#4245)

mingjiecn committed:
ENCD-6268 sex facet on donar matrix page (#4240)

gabdank committed:
ENCD-6278-qc-calculation (#4243)

keenangraham committed:
ENCD-6214-remove-data-collections (#4248)

gabdank committed:
ENCD-6231-hic-fragmentation (#4232)

forresttanaka committed:
ENCD-5983-page-and-carts-on-publication-pages (#4239)

emro committed:
ENCD-6259 improve immune cells page (#4251)

forresttanaka committed:
ENCD-6267-fcc-publication-filters-out-datapoint-experiments (#4244)

gabdank committed:
ENCD-6147 biosample calc (#4246)

khzlin committed:
ENCD-6279-add-output-type-accessibility (#4249)

jenjou committed:
ENCD-6274-rtqpcr (#4247)

forresttanaka committed:
ENCD-6280-series-files (#4252)

forresttanaka committed:
ENCD-6282-add-help-text-to-uniform-batch-growth-checkbox (#4254)

khzlin committed:
ENCD-6281-add-output-type-hic (#4253)

forresttanaka committed:
ENCD-6262-elements-cloning-and-elements-mappings-display (#4250)

emro committed:
ENCD-6277-update-valis-version-1.9 (#4256)

mingjiecn committed:
ENCD-5411 reverse sort publication by year(#4221)

emro committed:
ENCD-6284 sescc bug fix (#4259)

hitz committed:
update-version-v127.0 (#4260)

leepc12 committed:
ENCM-23-broken-annotation-ucsc-browser-stuff (#4262)

emro committed:
ENCM-26 update default encyc tracks (#4264)

keenangraham committed:
ENCM-29-pin-antlr (#4271)

ottojolanki committed:
ENCM-45-npm-timeout (#4274)

emro committed:
ENCM-39 replace view all links (#4275)

emro committed:
ENCM-25 add human body map to single cell (#4268)

emro committed:
ENCM-32 fix thymus thyroid mix-up (#4272)

jenjou committed:
ENCM-37-rna-qc-update (#4283)

jenjou committed:
ENCM-47 Descriptions added to DNase QC (#4279)

ian-whaling committed:
ENCM-42-reference-exp-input (#4277)

ottojolanki committed:
ENCM-40-dont-transform-limit-all (#4273)

KAndreeva committed:
ENCM-34 Updated GM reagent ID RegEx (#4280)

ian-whaling committed:
ENCM-36-update-HiC-QC (#4276)

jenjou committed:
ENCM-41-annotation-updates (#4278)

gabdank committed:
ENCM-43 supersedes donor (#4284)

gabdank committed:
ENCM-38 ChIA QC (#4281)

jenjou committed:
ENCM-50-chip-modern-audits (#4285)

keenangraham committed:
ENCM-54-pin-splinter (#4287)

jenjou committed:
ENCM-44-spatial-transcript-and-immune-ont-update (#4288)

jenjou committed:
ENCM-53-atac-chip-qc (#4286)

khzlin committed:
ENCM-59-bpnet-qc-object (#4292)

jenjou committed:
ENCM-52-long-read-output-types (#4289)

khzlin committed:
ENCM-57-post-differentiation-time-audit (#4294)

khzlin committed:
ENCM-58-sc-atac-qc-metrics-definitions (#4291)

jenjou committed:
ENCM-56-terms-for-e-g-mpra (#4295)

jenjou committed:
ENCM-62 Update home page card links to filter for released data only (#4296)

jenjou committed:
ENCM-55 Update File, Annotation, and Analysis Step for BPNet submission and update tests (#4293)

khzlin committed:
ENCM-61-add-encyclopedia-versions (#4298)

khzlin committed:
ENCM-60-introduce-genotyping (#4299)

gabdank committed:
update snovault 1.0.64 (#4300)

gabdank committed:
ENCM-64 aliases (#4303)

meenakshikagda committed:
ENCM-30-update-dsQTLs-to-caQTLs (#4302)

ian-whaling committed:
ENCM-63-hic-assay-title (#4301)

gabdank committed:
ENCM-65 indexing quality metrics (#4304)

jenjou committed:
ENCM-51-hic-audits (#4305)

gabdank committed:
update-snovault 1.0.65 (#4306)

forresttanaka committed:
ENCM-67-limiter-margin (#4307)

forresttanaka committed:
ENCM-68-body-margin (#4309)

forresttanaka committed:
ENCM-70-body-button (#4308)

jenjou committed:
ENCM-71-cl-owl (#4310)

keenangraham committed:
update-version-v128.0 (#4311)

forresttanaka committed:
ENCM-72-fix-adding-to-cart (#4312)

ottojolanki committed:
ENCM-75-gevent-error (#4314)

keenangraham committed:
ENCM-73-add-azure-url-to-metadata-tsv (#4313)

forresttanaka committed:
ENCM-27-show-only-released-results-for-cards-and-home-page-searches (#4315)

khzlin committed:
ENCM-77-cell-type-annotations (#4316)

forresttanaka committed:
ENCM-79-facet-count (#4318)

meenakshikagda committed:
ENCM-78-add-rfa-facet-human-donor (#4319)

khzlin committed:
ENCM-80-support-v40-biginteract (#4321)

jenjou committed:
ENCM-35-series-summary (#4322)

ian-whaling committed:
ENCM-82-hic-assay-title-fix (#4323)

forresttanaka committed:
ENCM-83 Improve the mouse development series page (#4327)

khzlin committed:
ENCM-85-add-degron-internal-tag (#4325)

gabdank committed:
ENCM-89-support-big-image-attachments (#4329)

khzlin committed:
ENCM-87-annotation-output-type (#4328)

jenjou committed:
ENCM-93 update ontology file (#4331)

forresttanaka committed:
ENCM-91-megamap-graph (#4332)

jenjou committed:
ENCM-95-exclude-series-audit (#4333)

forresttanaka committed:
ENCM-81 Enhance Multiomics Series Search Results (#4336)

khzlin committed:
ENCM-96-aggregate-series-related-dataset (#4334)

forresttanaka committed:
ENCM-84 Make a matrix for Degron experiments (#4335)

khzlin committed:
ENCM-97-file-output-type-update (#4337)

gabdank committed:
ENCM-100 update sno 1.0.66 (#4338)

gabdank committed:
update-version-v129.0 (#4342)

ottojolanki committed:
ENCM-106-docker-instructions

iyoungworth committed:
ENCM-105-add-output-variant-effect-quantifications (#4340)

khzlin committed:
ENCM-112-add-3d-modeling (#4347)

forresttanaka committed:
ENCM-83-mouse-dev-update (#4350)

emro committed:
ENCM-113 add human filter to home page link (#4352)

gabdank committed:
ENCM-120 hic-title-update (#4348)

gabdank committed:
ENCM-94 analyses in inserts (#4353)

emro committed:
ENCM-111 fix nerve button on body map (#4351)

jenjou committed:
ENCM-121 Exclude seqFISH in audit (#4349)

gabdank committed:
ENCM-118 add ENSEMBL V87 (#4354)

keenangraham committed:
SNO2-21-files-crash (#4356)

emro committed:
ENCM-119 fix organism dev error (#4355)

emro committed:
ENCM-114 mixed genome annotation crashes (#4357)

jenjou committed:
ENCM-124 add seqFISH to Pipelines (#4362)

mingjiecn committed:
ENCM-128-matrix-brain-term (#4366)

emro committed:
ENCM-88 fix crispr screens browser locations (#4361)

mingjiecn committed:
ENCM-125 add google analytics 4 (#4365)

forresttanaka committed:
ENCM-116 add disease series tab (#4367)

forresttanaka committed:
ENCM-115 Add FCC matrix (#4364)

meenakshikagda committed:
ENCM-130-reg-internal-tag (#4371)

emro committed:
ENCM-90 stem cell (#4372)

khzlin committed:
ENCM-129-add-output-type (#4370)

khzlin committed:
ENCM-127-add-bpnet-model (#4368)

khzlin committed:
ENCM-92-organoid-search-links (#4375)

khzlin committed:
ENCM-126-update-ccre-v4-agnostic (#4376)

khzlin committed:
ENCM-133-remove-walkme (#4378)

jenjou committed:
ENCM-131-encyclopedia-tags (#4374)

jenjou committed:
ENCM-132-bpnet-output-types (#4377)

gabdank committed:
ENCM-136 snovault 1.0.68 (#4380)

khzlin committed:
ENCM-137-undo-italicized-brackets (#4381)

forresttanaka committed:
ENCM-138-cart-facet-count (#4385)

mingjiecn committed:
ENCM-139-entex-link (#4382)

gabdank committed:
update-version-v130.0 (#4387)

jenjou committed:
ENCM-140-bpnet-normalized (#4388)

jenjou committed:
ENCM-148-annotation-type-pseudobulk (#4391)

khzlin committed:
ENCM-153-display-disease-annotation (#4392)

khzlin committed:
ENCM-158-annotation-treatment (#4393)

forresttanaka committed:
ENCM-162-valis-memory (#4396)

emro committed:
ENCM-141 add mouse body map to summary page (#4389)

forresttanaka committed:
ENCM-156 Add cCRE and Segway legends to Valis (#4395)

forresttanaka committed:
ENCM-165-fix-missing-files

iyoungworth committed:
ENCM-164-output-type-genic-features (#4398)

jenjou committed:
ENCM-160-snrna-seq-assay-title (#4394)

ottojolanki committed:
ENCM-171 Increase listen-backlog (#4399)

forresttanaka committed:
ENCM-144-rush-ad-cogdx (#4400)

gabdank committed:
update-version-v131.0 (#4401)

forresttanaka committed:
ENCM-167-aggregate-series-files (#4409)

jenjou committed:
ENCM-173-embryonic-fetal (#4405)

forresttanaka committed:
ENCM-168 Change order of Rush brain matrix tables (#4407)

forresttanaka committed:
ENCM-188-stem-cell (#4415)

forresttanaka committed:
ENCM-180-missing-cart-files (#4408)

jenjou committed:
ENCM-175-annotation-analysis-audit (#4404)

jenjou committed:
ENCM-152-publication-regex (#4410)

forresttanaka committed:
ENCM-176-long-target-facet (#4406)

leepc12 committed:
ENCM-170 (#4413)

jenjou committed:
ENCM-182-ann-type-enum (#4411)

jenjou committed:
ENCM-186-topic-modeling-enums (#4414)

jenjou committed:
ENCM-184-agg-series-summary-assay (#4412)

forresttanaka committed:
ENCM-189-series-crash (#4417)

Don’t assume the `related_datasets` property of a series object exists.

jenjou committed:
ENCM-193 restore audit inherit  (#4419)

gabdank committed:
update-version-v132.0 (#4418)

khzlin committed:
ENCM-197-ting-wang-support (#4420)

keenangraham committed:
ENCM-204-configure-shards-by-type (#4421)

khzlin committed:
ENCM-202-md5-restriction-uniform-processing (#4422)

keenangraham committed:
ENCM-209-reduce-local-indexing (#4424)

forresttanaka committed:
ENCM-195-multiomics-search (#4425)

shengchd committed:
ENCM-234-add-variant-funtion-pred (#4435)

* ENCM-234-add-variant-funtion-pred

* fix test

* fix test 2

* fix test 3

* fix bdd tests as ENCM-206

* add to analysis step

* update changelog

forresttanaka committed:
ENCM-206-single-cell-annotations (#4427)

ottojolanki committed:
update-version-v133.0

ottojolanki committed:
bump-version-v134.0 (#4437)

shengchd committed:
ENCM-236-add-snrna-enum (#4438)

forresttanaka committed:
ENCM-206-more-stuff (#4439)

stuartmiyasato committed:
ENCM-211-update-robots.txt (#4429)
